src/org/zaproxy/zap/session/CookieBasedSessionManagementHelper.java

```diff
@@ -61,6 +61,12 @@ public static void processMessageToMatchSession(HttpMessage message, List<HttpCo
 
 		// Make a copy of the session tokens set, as they will be modified
 		HttpSessionTokensSet tokensSet = session.getTokensNames();
+		
+		// If no tokens exists create dummy Object -> NPE
+		if (tokensSet == null) {
+			tokensSet = new HttpSessionTokensSet();
```

Comment:
Since it's empty it could just skip the following two loops (i.e. not being used at all).